pandora/src/components/SearchResults.jsx

```diff
@@ -18,7 +19,7 @@ const SearchResults = () => {
     };
 
     const handleFilter = () => {
-        dispatch(fetchProductsByUserChoices(categoryId, sort, filtre));
+        /* history.push(`/shop/${gender}/${categoryName}/${categoryId}/${sort || ""}/${filtre}/${limit || ""}/${offset || ""}`) */
```

Comment:
aslında filter için burası ad kullanılabilir. Url değiştiğin sayfa kendisini hesaplıyor sonuçta.